app/scripts/metamask-controller.js

```diff
@@ -399,6 +399,7 @@ export default class MetamaskController extends EventEmitter {
     this.loggingController = new LoggingController({
       messenger: this.controllerMessenger.getRestricted({
         name: 'LoggingController',
+        allowedActions: ['LoggingController:getState'],
```

Comment:
Hi @DDDDDanica, thanks for your work on this! I had two notes on conventions for instantiating messengers:

- Internal actions/events (i.e. in this case all message types prefixed with the `LoggingController:` namespace, including `LoggingController:getState`) should be excluded from a restricted messenger's allowlists. The idea is that internal message types are always supported by a messenger, so they don't have to be specified. 

- `LoggingControllerMessenger` is defined with empty allowlists for both external actions and events. For empty allowlists, we assign an empty array instead of omitting the property (if you're interested, here's some discussion on why we enforce this: https://github.com/MetaMask/core/issues/4033).

So in conclusion, all we need to do here is the following:

```suggestion
        allowedActions: [],
        allowedEvents: [],
```

Reply:
Hey @MajorLift thanks so much for reviewing and both notes make perfect sense !  I've pushed 2 empty arrays for `LoggingController` - [3443cc](https://github.com/MetaMask/metamask-extension/pull/27082/commits/3443cc7e22292809a3eb8dca40bbdc180938bc5b)

Reply:
Looks good!